.eslintrc.json

```diff
@@ -0,0 +1,51 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true,
+        "jest": true
+    },
+    "extends": [
+        "plugin:react/recommended",
+        "standard-with-typescript"
+    ],
+    "overrides": [
+        {
+          "files": ["*.jsx", "*.js", "*.ts", "*.tsx"]
+        }
+    ],
+    "ignorePatterns": ["*/*-env.d.ts"],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module",
+        "project": "./tsconfig.json"
+    },
+    "plugins": [
+        "react",
+        "@typescript-eslint"
+    ],
+    "rules": {
+        "semi": ["error", "always"],
+        "@typescript-eslint/semi": ["error", "always"],
+        "quotes": ["error", "double"],
```

Comment:
I would actually suggest double quotes 😅 I come from a C background where it really mattered, so it's grown to be my personal preference. In regards to @tylerpalmerdev's legitamate concern, we can have our developers modify their vscode settings so that it formats and lints on save. Lmk what you both think.